readthedocs/proxito/views/hosting.py

```diff
@@ -250,23 +253,47 @@ def _v0(self, project, version, build, filename):
                     "base_page": "",
                 },
                 "flyout": {
-                    "translations": [],
+                    "enabled": True,
```

Comment:
Note to myself: make all of these feature flags so we can enable/disable them based on projects and be able to disable them if we have to.